server/graphql/v2/interface/Account.js

```diff
@@ -169,8 +169,16 @@ const accountFieldsDefinition = () => ({
   },
   transferwise: {
     type: TransferWise,
-    resolve(collective) {
-      return collective;
+    async resolve(collective) {
+      const connectedAccount = await models.ConnectedAccount.findOne({
+        where: { service: 'transferwise', CollectiveId: collective.id, deletedAt: null },
```

Comment:
I don't think that you have to pass `deletedAt: null`, if `paranoid: false` is not added this is normally added by default.